modules/machineset-creating-azure-ultra-disk.adoc

```diff
@@ -0,0 +1,292 @@
+
+. Create a machine set using the updated configuration by running the following command:
++
+[source,terminal]
+----
+$ oc create -f <machine-set-name>.yaml
+----
+
+ifdef::pvc[]
+. Create a storage class that contains the following YAML definition:
++
+[source,yaml]
+----
+apiVersion: storage.k8s.io/v1
+kind: StorageClass
+metadata:
+  name: ultra-disk-sc <1>
+parameters:
+  cachingMode: None
+  diskIopsReadWrite: "2000" <2>
+  diskMbpsReadWrite: "320" <3>
+  kind: managed
+  skuname: UltraSSD_LRS
+provisioner: disk.csi.azure.com <4>
+reclaimPolicy: Delete
+volumeBindingMode: WaitForFirstConsumer <5>
+----
+<1> Specify the name of the storage class. This procedure uses `ultra-disk-sc` for this value.
+<2> Specify the number of IOPS for the storage class.
+<3> Specify the throughput in MBps for the storage class.
+<4> For Azure Kubernetes Service (AKS) version 1.21 or later, use `disk.csi.azure.com`. For earlier versions of AKS, use `kubernetes.io/azure-disk`.
+<5> Optional: Specify this parameter to wait for the creation of the pod that will use the disk.
+
+. Create a persistent volume claim (PVC) to reference the `ultra-disk-sc` storage class that contains the following YAML definition:
++
+[source,yaml]
+----
+apiVersion: v1
+kind: PersistentVolumeClaim
+metadata:
+  name: ultra-disk <1>
+spec:
+  accessModes:
+  - ReadWriteOnce
+  storageClassName: ultra-disk-sc <2>
+  resources:
+    requests:
+      storage: 4Gi <3>
+----
+<1> Specify the name of the PVC. This procedure uses `ultra-disk` for this value.
+<2> This PVC references the `ultra-disk-sc` storage class.
+<3> Specify the size for the storage class. The minimum is 4Gi.
+
+. Create a pod that contains the following YAML definition:
++
+[source,yaml]
+----
+apiVersion: v1
+kind: Pod
+metadata:
+  name: nginx-ultra
+spec:
+  nodeSelector:
+    disk: ultrassd <1>
+  containers:
+  - name: nginx-ultra
+    image: alpine:latest
+    command:
+      - "sleep"
+      - "infinity"
+    volumeMounts:
+    - mountPath: "/mnt/azure"
+      name: volume
+  volumes:
+    - name: volume
+      persistentVolumeClaim:
+        claimName: ultra-disk <2>
+----
+<1> Specify the label of the machine set that enables the use of ultra disks. This procedure uses `disk.ultrassd` for this value.
+<2> This pod references the `ultra-disk` PVC.
+endif::pvc[]
+
+.Verification
+
+#Would prefer to replace the two `oc` steps here with something from the web console. Otherwise, need some help understanding the `oc debug` step.#
+
+. Validate that the machines are created by running the following command:
++
+[source,terminal]
+----
+$ oc get machines
+----
++
+New machines should show the `Running` state.
+
+. Validate partition when machine is in running status and node is attached validate using below command for partition by running the following command:
++
+[source,terminal]
+----
+$ oc debug node/<node-name> -- chroot /host lsblk
+----
+
+#Is the below part of verification, or a next step?#
```

Comment:
Need some info here
Cc @damdo

Reply:
This is an example on how to make use of the newly attached UltraSSD from within a Pod. It's there as an example to explain how users can do that :)